crates/astria-conductor/src/executor.rs

```diff
@@ -292,11 +289,14 @@ impl<C: ExecutionClient> Executor<C> {
     ///
     /// This function returns an error if:
     /// - the call to the execution service's FinalizeBlock function fails
-    #[instrument(ret, err, skip_all, fields(execution_block_hash = hex::encode(&execution_block_hash), %sequencer_block_hash))]
+    // TODO: display the sequencer block hash bytes nicely
+    // #[instrument(ret, err, skip_all, fields(execution_block_hash =
```

Comment:
remove?

Reply:
Removed. All of this is covered in #165.